scripts/theia-version.js

```diff
@@ -0,0 +1,52 @@
+/********************************************************************************
+ * Copyright (C) 2019 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+/********************************************************************************
+ * Copyright (c) 2019 TypeFox and others
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ ********************************************************************************/
+
+ /* copied code from https://github.com/TypeFox/theia-xtext-sprotty-example/blob/master/theia/theia-version.js#L1-26 */
+
+const shell = require('shelljs')
```

Comment:
When running the command `yarn run publish:latest` I get the following error:

```
yarn publish:latest
yarn run v1.17.3
$ node scripts/theia-version.js latest && lerna publish && yarn publish:check
internal/modules/cjs/loader.js:584
    throw err;
    ^

Error: Cannot find module 'shelljs'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/Users/vincentfugnitto/workspace/theia-cpp-extensions/scripts/theia-version.js:28:15)
```

A `devDependency` to **shelljs** is likely required.

Reply:
Since **shelljs** is not a dependency present under the main repository, I believe it will
require a new CQ.

Reply:
I guess it should be added in package.json? It would be a dev-dependency, which has a different procedure for CQ: 
https://github.com/eclipse-theia/theia/wiki/Registering-CQs#case-new-or-updated-testbuild-dependency-eg-devdependencies

Reply:
Yes exactly, a `devDependency` under the root's `package.json`.
Great that the procedure is different!

Reply:
Registered a CQ for shelljs
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=20969